app.js

```diff
@@ -4,11 +4,17 @@ const express = require('express');
 const path = require('path');
 const cookieParser = require('cookie-parser');
 const logger = require('morgan');
+const connectDB = require('./boostrap/database');
 
 const indexRouter = require('./routes/index');
 const weatherRouter = require('./routes/weather');
+const errorHandler = require('./middleware/errorHandler');
 
 const app = express();
+connectDB();
```

Comment:
connectDB returns a promise so we need to either use callback on it  or use await.

Reply:
Yahh, :D that's true, fixed thanks